app/js/components/orderEntry/addForm/DraftDataTable.jsx

```diff
@@ -1,88 +1,122 @@
+        {frequency && ` ${frequency},`}
+        {route && ` ${route}`}
+        {duration && `, for ${duration} ${durationUnit} total`}
+        {reason && `, (Reason: ${reason})`}
+        {drugInstructions && ` (${drugInstructions})`}
+        {dispensingQuantity && ` (Dispense: ${dispensingQuantity} ${dispensingUnit})`}
+      </p>
+    );
+
+    const discontinueForm = (
+      <div>
+        <p> Discontinue {drugName} </p>
+        <p className="left label-margin">
+          <label name="reason">For</label>
+        </p>
+        <p className="left">
+          <input
+            name="reason"
+            placeholder="reason"
+            id="discontinueReason"
+            type="text"
+            size="50" />
+        </p>
+      </div>
+    );
+
+    return (
+      <tr key={orderNumber}>
+        <td>{action}</td>
+        <td >
+          {action === "DISCONTINUE" ?
+            discontinueForm :
+            details
+          }
+        </td>
+        <td>
+          <a href="#" >
+            <i className="icon-edit" title="Edit" />
+          </a>
+          <a href="#" onClick={() => handleDiscardOneOrder(order)}>
+            <i className="icon-remove" title="Delete" />
+          </a>
+        </td>
+      </tr>
+    );
+  })
+
+
+  render() {
+    const { draftOrders, handleDiscardAllOrders } = this.props;
+
+    return (
+      <div className="draft-spacing ">
+        <h2>Unsaved Draft Orders ({draftOrders.length})</h2>
+        <table className="table bordered mw-958-px">
+          <thead>
+            <tr>
+              <th className="w-120-px">Status</th>
+              <th>Details</th>
+              <th className="w-81-px">Actions</th>
             </tr>
-          );
-        })}
-      </tbody>
-    </table>
-    <br />
-    <input
-      type="button"
-      onClick={handleCancel}
-      className="button cancel"
-      value={draftOrders.length > 1 ? "Discard All" : "Discard"}
-    />
-    <input
-      type="submit"
-      onClick={handleSubmit}
-      className="button confirm right"
-      value="Sign and Save"
-    />
-  </div>
-);
+          </thead>
+          <tbody>
+            {this.showOrders(draftOrders)}
+          </tbody>
+        </table>
+        <br />
+        <input
+          type="button"
+          onClick={handleDiscardAllOrders}
+          className="button cancel"
+          value={draftOrders.length > 1 ? "Discard All" : "Discard"}
+        />
+        <input
+          type="submit"
+
+          className="button confirm right"
+          value="Sign and Save"
+        />
+      </div>
+    );
+  }
+}
+
 
 DraftDataTable.propTypes = {
-  draftOrders: PropTypes.array.isRequired,
-  handleCancel: PropTypes.func.isRequired,
-  handleSubmit: PropTypes.func.isRequired,
-  handleEditDrugOrder: PropTypes.func.isRequired,
+  draftOrders: PropTypes.arrayOf(PropTypes.any).isRequired,
+  handleDiscardAllOrders: PropTypes.func.isRequired,
+  // handleSubmit: PropTypes.func.isRequired,
```

Comment:
Could you remove the commented out code if they are not needed?

Reply:
they will be needed for the next task, that's why I have left them

Reply:
Alright.

Reply:
If they are not for this task, why are they here in the first place?

Reply:
The draft table was previously implemented, and this was part of the implementation, in refactoring how the draft table works, I handled functions that were more global to the scope of the draft table,  but for the ```handleEditDratOrder```, @Betty-Kebenei  will need to refactor her implementation of how the function works, which will be done once this PR is merged. As for the submit function, I will need to sync with part of the team for it to be implemented appropriately.

Reply:
If it is not part of what is required for this pull request, then committing it while commented out does not seem like a good practice.

Reply:
okay then, I'll remove them